.swiftlint.yml

```diff
@@ -89,7 +94,6 @@ opt_in_rules:
   - switch_case_on_newline
   - test_case_accessibility
   - toggle_bool
-  - type_contents_order
```

Comment:
`type_contents_order` does not support a dedicated SwiftUI `body` order type, so its OK to move it to disabled section:
 
<img width="1486" alt="Screenshot 2025-06-18 at 16 32 33" src="https://github.com/user-attachments/assets/e0f98cb4-0ad3-4284-8dfb-fd7e8eed0ecc" />